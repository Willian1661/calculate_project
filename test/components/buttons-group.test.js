const button = require("../../src/components/button");
const buttonsGroup = require("../../src/components/buttons-group")

describe("pattern div component", () =>{

    it("is passing dynamic buttons", () => {
        const buttons =[ 
        button("foo","fooClass"),
        button("baz","bazClass")
    ]
        expect(buttonsGroup("buttonsId",buttons)).toMatchSnapshot()
    });
	
})