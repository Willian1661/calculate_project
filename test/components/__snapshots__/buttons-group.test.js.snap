// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pattern div component is passing dynamic buttons 1`] = `
"
		<div id="buttons-id">
			<button class="fooClass">foo</button>
			<button class="bazClass">baz</button>}
		</div>"
`;
